docs/docs/guides/wallet/transactions.md

```diff
@@ -7,6 +7,124 @@ sidebar_label: 'Tutorial: Sending Transactions'
 
 This guide provides insights into sending transactions using web3.js, covering various scenarios from utilizing a local wallet to sending raw transactions.
 
+## Transaction Type 0 (Legacy)
+
+A Legacy Transaction refers to a transaction that was created using an older version of Ethereum's transaction format, also known as "transaction type 0". This transaction format was used before the EIP-1559 upgrade, which was implemented in August 2021.
+
+```ts
+import { Web3 } from"web3";
+
+const web3 = new Web3("https://rpc2.sepolia.org");
+
+async function txLegacy() {
+  const wallet = web3.eth.wallet.add("YOUR_PRIVATE_KEY"); //make sure you have funds
+
+  const sender = wallet[0].address;
+  const recipient = "0x807BFe4940016B5a7FdA19482042917B02e68359";
+  const value = 1; //wei
+  const nonce = await web3.eth.getTransactionCount(sender);
+  const gas = 21000;
+  const gasPrice = await web3.eth.getGasPrice();
+
+  const tx = {
+    from: sender,
+    to: recipient,
+    value,
+    nonce,
+    gas,
+    gasPrice,
+  };
```

Comment:
We have defaultTransactionType: '0x2'

So if you want to sent legacy tx. set type: 0